packages/vue-renderer/src/renderer.js

```diff
@@ -284,6 +286,11 @@ export default class VueRenderer {
       renderContext.modern = modernMode === 'client' || isModernRequest(req, modernMode)
     }
 
+    // renderContext.static (old _generate)
```

Comment:
Maybe we should add a TODO here?
```suggestion
    // renderContext.static (old _generate) TODO: Remove in Nuxt 3
```